### PR TITLE
Release v0.46.0

### PR DESCRIPTION
**Pull request by Farhad-Shabani**
Closes: #XXX

______

### PR author checklist:
- [X] Added changelog entry, using [`unclog`](https://github.com/informalsystems/unclog).
- [ ] Added tests.
- [X] Linked to GitHub issue.
- [X] Updated code comments and documentation (e.g., `docs/`).
- [X] Tagged *one* reviewer who will be the one responsible for shepherding this PR.

### Reviewer checklist:

- [ ] Reviewed `Files changed` in the GitHub PR explorer.
- [ ] Manually tested (in case integration/unit/mock tests are absent).

### CODE REVIEW

File: crates/ibc-query/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name         = "ibc-query"
-version      = "0.45.0"
+version      = "0.46.0"
```
Comment (rnbguy) [inline crates/ibc-query/Cargo.toml:3]: Question. How are we distributing `ibc-query` package? If it is a separate package, do we still need to keep the same version number as `ibc`? (e.g. `ibc-derive` has different version number)

**Review by rnbguy — COMMENTED** [review]
Looks good. Just a nit request. Is it okay to make the `.toml` files ending with newlines?

File: crates/ibc-query/Cargo.toml (hunk 1 above)
Comment (Farhad-Shabani) [reply crates/ibc-query/Cargo.toml:3]: Good question!

More of a common practice, and in a simplistic view `ibc-query` is dependent on `ibc` and this approach ensures that all dependencies are kept in sync. Also, for an easier map version for users.

As for the `ibc-derive` things are a bit different. That’s not directly dependent on `ibc` crate. we re-export everything in `ibc-derive`, so projects won’t need to explicitly have it as a dependency. Also, `ibc-derive` only needs to get updated if there are only changes to `ClientState` APIs.

Comment (rnbguy) [reply crates/ibc-query/Cargo.toml:3]: Sounds good to me. Just wanted to make sure you're aware of this. Thanks :raised_hands:

**Review by rnbguy — APPROVED** [review]
Thanks @Farhad-Shabani :tada:

**Pull request merged**